website/snippets/auth0-uri.md

```diff
@@ -1,11 +1,13 @@
 
 The URI used for SSO connections on multi-tenant dbt Cloud instances will vary based on your dbt Cloud hosted region. Use your login URL (also referred to as your Access URL) to determine the correct Auth0 URI for your environment.
```

Comment:
This is not for multi-tenants only. Even single-tenants will be using this table.